specification.markdown

```diff
@@ -778,3 +774,47 @@ minOccurs: 0
 type: Binary
 
 description: Used to void damaged data, to avoid unexpected behaviors when using damaged data. The content is discarded. Also used to reserve space in a sub-element for later use.
+
+# IANA Considerations
+
+This document creates a new IANA Registry called
+"CELLAR EBML Element ID Registry".
+
+Element IDs are described in section {{#element-id}}.  Element IDs are
+encoded using the VINT mechanism described in
+section {{#variable-sized-integer}} can be between one and five bytes
+long. Five byte long Element IDs are possible only if declared in the header.
+
+One byte Element IDs are numbers between 1 and 126. These items are valuable
```

Comment:
Maybe better to say something like "The VINT_DATA of one-byte Element IDs are numbers between 1 and 126." since the number references only the VINT_DATA part. Otherwise a one byte Element ID would only be in the range of 0x81...0xFE.